src/version.rs

```diff
@@ -88,6 +88,13 @@ impl From<&(u32, u32, u32)> for SemanticVersion {
     }
 }
 
+// Convert an &version into a version.
+impl From<&SemanticVersion> for SemanticVersion {
```

Comment:
Could this introduce stealthy clones in future code?

Reply:
Good thought. In this case I think it is ok.
For one thing `.from()` and `.into()` should probably always be thought of as "potentially as costly as a clone".
For another this is not generic, and we know the cost to clone a `SemanticVersion` is small.
What do you think?